src/software/geom/util.cpp

```diff
@@ -104,9 +104,9 @@ double dist(const Point &first, const Rectangle &second)
     return *std::min_element(distances.begin(), distances.end());
 }
 
-double distsq(const Vector &first, const Segment &second)
+double distsq(const Point &first, const Segment &second)
```

Comment:
If we are going to avoid using shortened names, might as well change this to `distanceSquared` as well? But maybe better kept to another PR, don't want this PR getting any bigger 😅

Reply:
Agree, this could be fixed when we port over ```geom/util.cpp``` to ```new_geom```

Reply:
Agree, this could be fixed when we adapt ```geom/util.cpp``` to ```new_geom```

Reply:
Added as criteria for #959